app/forms/idv/phone_form.rb

```diff
@@ -59,7 +61,11 @@ def determine_initial_values(international_code: nil, phone: nil)
         international_code ||= country_code_for(phone)
       end
 
-      phone = PhoneFormatter.format(phone, country_code: international_code)
+      if failed_phone_numbers.include?(Phonelib.parse(phone).e164)
+        phone = nil
+      else
+        phone = PhoneFormatter.format(phone, country_code: international_code)
+      end
 
       [international_code, phone]
```

Comment:
Should we also avoid setting the international code associated with the phone number if we're blanking it out?

```suggestion
      if failed_phone_numbers.include?(Phonelib.parse(phone).e164)
        [nil, nil]
      else
        phone = PhoneFormatter.format(phone, country_code: international_code)
        [international_code, phone]
      end
```